.github/workflows/verify-this-repo.yml

```diff
@@ -8,6 +8,3 @@ on:
 jobs:
   call-verify-workflow:
     uses: Lombiq/GitHub-Actions/.github/workflows/verify-submodule-pull-request.yml@dev
-  
-  call-check-merge-conflict:
-    uses: Lombiq/GitHub-Actions/.github/workflows/check-merge-conflict.yml@issue/OSOE-144
```

Comment:
This is not run for this repo now. Keep it here, just rename this workflow like in OSOCE.